fastserde/avro-fastserde/src/main/java/com/linkedin/avro/fastserde/BufferBackedPrimitiveFloatList.java

```diff
@@ -61,6 +65,15 @@ public BufferBackedPrimitiveFloatList(Collection<Float> c) {
     byteBuffer = new CompositeByteBuffer(c != null);
   }
 
+  // For testing purpose
```

Comment:
I suggest using JavaDoc rather than a double slash, and mentioning that this is a shallow copy.